api/v1/utils/events.js

```diff
@@ -1,10 +1,14 @@
 var _ = require('lodash');
-var ALL_TAGS = ['HACKATHON', 'SCHEDULE'];
+var ALL_TAGS = ['HACKATHON', 'SCHEDULE', 'ANNOUNCEMENT'];
```

Comment:
Let's get rid of all of these announcements additions. We have the announcements endpoints for that